bci_essentials/classification/ssvep_basic_tf_classifier.py

```diff
@@ -26,7 +26,7 @@ class SSVEP_basic_tf_classifier(Generic_classifier):
 
     """
 
-    def set_ssvep_settings(self, sampling_freq, target_freqs):
+    def set_ssvep_settings(self, sampling_freq: int, target_freqs: tuple[int]) -> None:
```

Comment:
I think target_freqs is currently stored as list.

Reply:
You're right, I'll fix this. Thanks!

Reply:
Ok @brianjohannes, this has been updated for this file. In ssvep_riemmanian_mdm_classifier.py target_freqs is defined as a np.ndarray. Should this also be a list[int]?

Reply:
Ahhh sorry I just saw this now, it's a list[float] here, because the frequency values are not necessarily whole numbers. In ssvep_riemannian_mdm_classifier it can stay as np.ndarray, because once the classifier is trained then we can no longer change the frequency values, so there is no need for it to be mutable anymore.